make/test/JtregNativeHotspot.gmk

```diff
@@ -852,6 +852,9 @@ ifeq ($(call isTargetOs, linux), true)
   BUILD_HOTSPOT_JTREG_EXECUTABLES_LIBS_exeFPRegs := -ldl
   BUILD_HOTSPOT_JTREG_LIBRARIES_LIBS_libAsyncGetCallTraceTest := -ldl
   BUILD_HOTSPOT_JTREG_LIBRARIES_LDFLAGS_libfast-math := -ffast-math
+# SapMachine RS 2023-10-04
```

Comment:
Should rather be something like
`# SapMachine 2023-10-04 New malloc trace implementation`
So, without the initials and with some description